packages/react/src/Timeline/Timeline.module.css

```diff
@@ -0,0 +1,62 @@
+.Timeline {
+  --timelineSpacing: var(--base-size-16);
+  --itemLineHeight: var(--brand-text-lineHeight-300);
+  --bulletSize: var(--base-size-8);
+  --bulletSpacing: 2px;
+  --bulletY: calc(var(--itemLineHeight) / 2 - var(--bulletSize) / 2);
+  --lineWidth: 1.5px;
+
+  margin: 0;
+  padding: 0px;
+  list-style: none;
+}
+
+.Timeline-item {
+  position: relative;
+  max-width: 44ch;
+  padding-inline-start: calc(var(--bulletSize) + var(--timelineSpacing));
+  color: var(--brand-color-text-default);
+  font-family: var(--brand-body-fontFamily);
+  font-size: var(--brand-text-size-300);
+  font-weight: var(--brand-text-weight-200);
+  line-height: var(--itemLineHeight);
+}
+
+.Timeline-item:not(:last-child) {
+  padding-bottom: var(--base-size-32);
+}
+
+.Timeline-item::before,
+.Timeline-item::after {
+  content: '';
+  position: absolute;
+}
+
+.Timeline-item::before {
+  width: var(--bulletSize);
+  height: var(--bulletSize);
+  top: var(--bulletY);
+  left: 0;
+  border: solid var(--lineWidth) var(--brand-color-border-default);
```

Comment:
The line art is really faint for me. I think anyone using a monitor with lowered brightness / poorer contrast like mine may not even notice it. Could we darken / brighten (depending on color mode) the value of the color here? I know these match the site refactor designs, so mostly calling it out as general feedback.

![Screenshot 2023-08-07 at 14 23 24](https://github.com/primer/brand/assets/13340707/45d0f877-28d0-48f8-99cd-de48b7ff1fe1)

Reply:
That's a good point. As the timeline is a critical part of visually understanding the pattern though, could we use component-specific tokens for now? Changing global values for border-default is likely going to be a long and sweeping change that can be done separately to this PR.